cumulusci/tasks/dx_convert_from.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+from cumulusci.tasks.sfdx import SFDXBaseTask
+
+
+class DxConvertFrom(SFDXBaseTask):
+    task_options = {
+        "command": {
+            "description": "The full command to run with the sfdx cli.",
+            "required": True,
+        },
+        "extra": {"description": "Append additional options to the command"},
+        "src_dir": {
+            "description": "The path to the src directory where converted contents will be stored. Defaults to src/"
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+        if "src_dir" not in self.options:
+            self.options["src_dir"] = "src"
+
+        # append the -d option to sfdx force:source:convert
+        self.options[
+            "command"
+        ] = f"{self.options['command']} -d {self.options['src_dir']}"
+
+    def _run_task(self):
+        src_dir = Path(self.options["src_dir"])
+        if src_dir.exists():
+            self._clear_directory(src_dir)
```

Comment:
You can use [shutil.rmtree](https://docs.python.org/3/library/shutil.html#shutil.rmtree) for this (I _think_ the sfdx command will create the src directory if it doesn't exist)